src/main/java/org/opensearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -206,10 +206,10 @@
     public static final String LEGACY_OPENDISTRO_AD_BASE_URI = LEGACY_AD_BASE + "/detectors";
     public static final String AD_BASE_URI = "/_plugins/_anomaly_detection";
     public static final String AD_BASE_DETECTORS_URI = AD_BASE_URI + "/detectors";
-    public static final String AD_THREAD_POOL_PREFIX = "opendistro.ad.";
+    public static final String AD_THREAD_POOL_PREFIX = "opensearch.ad.";
     public static final String AD_THREAD_POOL_NAME = "ad-threadpool";
     public static final String AD_BATCH_TASK_THREAD_POOL_NAME = "ad-batch-task-threadpool";
-    public static final String AD_JOB_TYPE = "opendistro_anomaly_detector";
+    public static final String AD_JOB_TYPE = "opensearch_anomaly_detector";
```

Comment:
I think this might be a problem. 
Job Scheduler uses this identifier to understand what job it is. 
If an old index of job scheduler exists from ODFE, and the cluster is upgraded to OpenSearch, the new job scheduler might not find the right place to run the job for AD. 
Adding @ylwu-amzn to the conversation.

Reply:
Thank you for the comments!
I have submitted another commit to change this job type back.

Reply:
Thanks @spbjss!